pkg/controllers/jivavolume_controller.go

```diff
@@ -38,7 +38,6 @@ import (
 	operr "github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	appsv1 "k8s.io/api/apps/v1"
-	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/api/core/v1"
```

Comment:
@asquare14 can we use `corev1` alias and remove the `v1 "k8s.io/api/core/v1"` , generally we prefer to have `corev1` as alias to avoid the confusion b/w various `v1` k8s apis.

Reply:
Done!